pluton/harness/run.go

```diff
@@ -45,6 +47,38 @@ func RunSuite(pattern string) {
 	}
 	suite := harness.NewSuite(opts, tests)
 
+	// setup a node for in cluster services not tied to individual test life-cycles
+	switch Opts.CloudPlatform {
+	case "gce":
+		var bastionDir = filepath.Join(Opts.OutputDir, "bastion")
+
+		if err := os.MkdirAll(bastionDir, 0777); err != nil {
+			fmt.Printf("setting up bastion cluster: %v\n", err)
+			os.Exit(1)
+		}
+
+		// override machine size for bastion
+		savedVal := Opts.GCEOptions.MachineType
```

Comment:
Since options is passed to NewCluster there is no need to use the global, just make a copy and edit that instead.